src/analyses/constant_propagator.cpp

```diff
@@ -566,12 +572,12 @@ bool constant_propagator_domaint::partial_evaluate_with_all_rounding_modes(
   exprt first_result;
   for(std::size_t i = 0; i < rounding_modes.size(); ++i)
   {
-    constant_propagator_domaint child(*this);
-    child.values.set_to(
+    valuest beacon_values = known_values;
```

Comment:
Beacon? Haven't heard that term before

Reply:
Should I just use `tmp_values`? I'll go with that for the moment, all ideas appreciated.

Reply:
Either that or comment to say what a beacon is in this context :) I assume that's a term used elsewhere, I just haven't heard of it

Reply:
That was just me making things up with my limited command of the language.